ros2_odometry_node:
  ros__parameters:
    drive_type: "MECANUM"  # Buraya string giriyoruz, kod enum'a çeviriyor
    
    # Mecanum'a özel parametreler
    ticks_per_rev: 4200.0
    wheel_radius: 0.04
    robot_l: 0.20
    robot_w: 0.23
    angle_epsilon: 0.0001